pkg/controller/networkpolicy/validate_test.go

```diff
@@ -1528,14 +1616,236 @@ func TestValidateAntreaGroup(t *testing.T) {
 					},
 				},
 			},
+			operation:      admv1.Create,
 			expectedReason: "At most one of podSelector, externalEntitySelector, serviceReference, ipBlocks or childGroups can be set for a Group",
 		},
+		{
+			name: "anp-group-set-with-podselector-specified-to-update",
```

Comment:
hard to understand the case from the name, you probably mean "update anp-group-set-with-podselector-specified"